client-js/app/elements/labrad-registry.ts

```diff
@@ -95,6 +105,7 @@ export class LabradRegistry extends polymer.Base {
     for (var j in resp.keys) {
       this.push('keys', {name: resp.keys[j], value: resp.vals[j]});
     }
+    this.$.combinedList.selected = null;
```

Comment:
Can you do this by assigning to `this.idx` instead of touching the DOM directly?